src/report_modules/report_module_github_ci/src/report_format_github_ci.cpp

```diff
@@ -0,0 +1,223 @@
+/*
+ * Copyright 2023 CARIAD SE.
+ * Copyright 2024 BMW AG
+ *
+ * This Source Code Form is subject to the terms of the Mozilla
+ * Public License, v. 2.0. If a copy of the MPL was not distributed
+ * with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+#include "report_format_github_ci.h"
+#include "common/config_format/c_configuration.h"
+#include "common/config_format/c_configuration_report_module.h"
+#include "common/result_format/c_checker.h"
+#include "common/result_format/c_checker_bundle.h"
+#include "common/result_format/c_domain_specific_info.h"
+#include "common/result_format/c_issue.h"
+#include "common/result_format/c_locations_container.h"
+#include "common/result_format/c_metadata.h"
+#include "common/result_format/c_parameter_container.h"
+#include "common/result_format/c_result_container.h"
+#include "common/result_format/c_xml_location.h"
+#include "stdafx.h"
+
+#include "common/qc4openx_filesystem.h"
+#include <set>
+
+XERCES_CPP_NAMESPACE_USE
+
+static std::map<eIssueLevel, std::string> mapIssueLevelToString = {{eIssueLevel::INFO_LVL, "::notice::"},
+                                                                   {eIssueLevel::WARNING_LVL, "::warning::"},
+                                                                   {eIssueLevel::ERROR_LVL, "::error::"}};
+
+// Main Programm
+int main(int argc, char *argv[])
+{
+    const std::string strToolpath(argv[0]);
+
+    if (argc != 2)
+    {
+        ShowHelp(strToolpath);
+
+        if (argc < 2)
+            return 0;
+        else
+            return -1;
+    }
+
+    std::string strFilepath = argv[1];
+    struct stat fileStatus;
+
+    XMLPlatformUtils::Initialize();
+
+    cParameterContainer inputParams;
+
+    // Default parameters
+    inputParams.SetParam("strInputFile", "Result.xqar");
+
+    if (StringEndsWith(ToLower(strFilepath), ".xqar"))
+    {
+        if (stat(strFilepath.c_str(), &fileStatus) == -1) // ==0 ok; ==-1 error
```

Comment:
The inluce on line 23 is importing `std::filesystem`. Maybe we can use [`std::filesystem::exists`](https://en.cppreference.com/w/cpp/filesystem/exists) here for consistency?

Reply:
done.